pages/storage.vue

```diff
@@ -121,19 +121,23 @@ export default {
   },
   async asyncData({ $content }) {
     const index = await $content(
-      'services/file-storage-and-transfer/index'
+      'meta', 'category', 'services', 'file-storage-and-transfer'
     ).fetch();
```

Comment:
this should have it's own header data, add that to the tool yaml instead of pulling from this one

Reply:
should we delete the meta/category/services/file-storage-and-transfer file in the content repo then?

Reply:
if it isn't used anywhere else, yes - but I think there is also a service page for storage, so maybe hold off until that's fully settled (cc @paulstey )